.merlin

```diff
@@ -1,52 +0,0 @@
-FLG -rectypes -thread -safe-string -w +a-4-9-27-41-42-44-45-48-50
-
-S clib
-B clib
-S config
-B config
-S lib
-B lib
-S kernel
-B kernel
-S kernel/byterun
-B kernel/byterun
-S library
-B library
-S engine
-B engine
-S pretyping
-B pretyping
-S interp
-B interp
-S proofs
-B proofs
-S tactics
-B tactics
-S printing
-B printing
-S parsing
-B parsing
-S stm
-B stm
-S vernac
-B vernac
-S toplevel
-B toplevel
-S plugins/ltac
-B plugins/ltac
-S API
-B API
-S ide
-B ide
-
-S tools
-B tools
-S tools/coqdoc
-B tools/coqdoc
-S dev
-B dev
-
-S plugins/**
-B plugins/**
-
-PKG threads.posix camlp5
```

Comment:
Why remove this?

Reply:
I think the answer is in the PR description: dune automatically generates the `.merlin` file.